### PR TITLE
add prometheus agent mode installplan

### PR DESCRIPTION
**Pull request by bpschmitt**
# Summary

Adding an `installPlan` for Prometheus Agent Mode.

## Pre merge checklist

- [X] Did you include an InstallPlan and Documentation reference?
- [X] Are all documentation links publically accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [X] Did you check your descriptive content for spelling and grammar errors?

### CODE REVIEW

File: install/prometheus-agent-mode/install.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+description: Use the Prometheus Agent to scrape Prometheus endpoints in your Kubernetes cluster and send metrics to New Relic using Prometheus remote write.
+
+target:
+  type: integration
+  destination: kubernetes
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com
```
Comment (mangulonr) [inline install/prometheus-agent-mode/install.yml:13]: The final URL is available (temporary empty doc)
https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/

File: install/prometheus-agent-mode/install.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+description: Use the Prometheus Agent to scrape Prometheus endpoints in your Kubernetes cluster and send metrics to New Relic using Prometheus remote write.
+
+target:
+  type: integration
+  destination: kubernetes
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent/
```
Comment (aswanson-nr) [inline install/prometheus-agent-mode/install.yml:13]: This doc link is an empty page, we'll just need that to be live before merging

**Review by aswanson-nr — CHANGES_REQUESTED** [review]

File: install/prometheus-agent-mode/install.yml (hunk 2 above)
Comment (mangulonr) [reply install/prometheus-agent-mode/install.yml:13]: Understood. Next week we should meet in order to define the deployment plan.
But the order should be:

* Docs deployed
* Merging this PR
* Merging Quickstarts PRs (7 new integrations + Prometheus Agent) which are referencing the install plan of this PR

cc @seemantk @bpschmitt @thezackm

**Review by josephgregoryii — APPROVED** [review]

**Review by aswanson-nr — APPROVED** [review]
The docs link will be available before anything that uses this install plan is available, so it's fine to get this merged.

**Pull request merged**